prime-dotnet-webapi/Configuration/Database/Seeded Tables/StatusReasonConfiguration.cs

```diff
@@ -26,7 +26,7 @@ public override IEnumerable<StatusReason> SeedData
                     new StatusReason { Code = (int)StatusReasonType.IdentityProvider,             Name = "User authenticated with a method other than BC Services Card"                   },
                     new StatusReason { Code = (int)StatusReasonType.RequestingRemoteAccess,       Name = "User has Requested Remote Access"                                               },
                     new StatusReason { Code = (int)StatusReasonType.NoAssignedAgreement,          Name = "Terms of Access to be determined by an Adjudicator"                             },
-                    new StatusReason { Code = (int)StatusReasonType.NoVerifiedAddress,            Name = "No address from BCSC. Enrollee entered address."                                },
+                    new StatusReason { Code = (int)StatusReasonType.NoVerifiedAddress,            Name = "No address from BC Service Card. Enrollee entered address."                     },
```

Comment:
I don't _think_ this is necessary (but is, admittedly, very thorough). We are primarily concerned with unifying the messaging to the Enrollees and not so much with the system-level messages to the adjudicators

Reply:
I was going by the ticket where it said the entire prime app shouldn't have any abbreviation. Also confirmed it with Jason before committing that change.